Networking/Networking/Model/Product/ProductVariation.swift

```diff
@@ -181,7 +181,7 @@ public struct ProductVariation: Decodable {
         let shippingClass = try container.decodeIfPresent(String.self, forKey: .shippingClass)
         let shippingClassID = try container.decode(Int.self, forKey: .shippingClassID)
 
-        let image = try container.decode(ProductImage.self, forKey: .image)
+        let image = try container.decodeIfPresent(ProductImage.self, forKey: .image)
```

Comment:
Just thinking out loud here... it might make sense to default to a placeholder if the `image` key is not present, but probably not here, as that seems like a concern of the UI layer. Maybe an extension to the Yosemite corresponding model object, declared in the UI layer would make sense later on? Maybe in Yosemite's `ProductVariation+ReadOnly` `createReadOnlyImage`?

Reply:
@ctarda — interesting thought. We do have placeholder images [defined here](https://github.com/woocommerce/woocommerce-ios/blob/2e3d54d52e82fba938f583479090cdadad972982/WooCommerce/Classes/Extensions/UIImage%2BWoo.swift), and (as of right now) the individual UI elements manage their display (uaully via `WordPressUI`'s `ImageView` extensions. Feel free to open a new issue if you would like to explore this more.

>   but probably not here, as that seems like a concern of the UI layer

agree 😄